app/script/media/MediaEmbeds.js

```diff
@@ -89,10 +89,12 @@ z.media.MediaEmbeds = (function() {
    * @returns {string} Youtube embed URL
   */
   const _generate_youtube_embed_url = function(url) {
-    if (url.match(/youtu.be|youtube.com/)) {
+    if (url.match(_regex.youtube)) {
 
       const video_id = url.match(/(?:embed\/|v=|v\/|be\/)([a-zA-Z0-9_-]{11})/);
-      if (!video_id) return;
+      if (!video_id) {
+          return;
```

Comment:
you should get a plugin for editorconfig for your editor: http://editorconfig.org/